hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/ActionStatusUpdateHandlerService.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa;
+
+import java.util.List;
+import org.eclipse.hawkbit.repository.ControllerManagement;
+import org.eclipse.hawkbit.repository.EntityFactory;
+import org.eclipse.hawkbit.repository.builder.ActionStatusCreate;
+import org.eclipse.hawkbit.repository.event.remote.ActionStatusUpdateEvent;
+import org.eclipse.hawkbit.repository.model.Action;
+import org.eclipse.hawkbit.repository.model.Action.Status;
+import org.eclipse.hawkbit.security.SystemSecurityContext;
+import org.springframework.context.event.EventListener;
+
+/**
+ * A service that listens and processes the TargetStatusForDistributionSetEvent
```

Comment:
```suggestion
 * A service that listens and processes the ActionStatusUpdateEvent
```
Maybe you could add a JavaDoc link to the class

Reply:
JavaDoc link added